python/ray/tests/test_runtime_env.py

```diff
@@ -43,7 +43,7 @@
 def test_get_wheel_filename():
     """Test the code that generates the filenames of the `latest` wheels."""
     # NOTE: These should not be changed for releases.
-    ray_version = "2.3.0rc0"
+    ray_version = "3.0.0.dev0"
```

Comment:
Can we break it apart so it's not caught in the mass-rename?

For reference, these are the steps the release manager does to rename. I think it's a bad idea to make a manual exception here.
```bash
# this updates versions in python and c++ files (This should, at a minimum, update python/ray/init.py, [build-docker.sh](http://build-docker.sh/), and src/ray/common/constants.h). (changes from python/ray/_private/utils.py, python/ray/tests/test_runtime_env.py, dashboard/client/package-lock.json don't need to be committed)
grep -l -R "3.0.0.dev0" * | grep -v "yaml" | grep -v "rst" | xargs -n 1 sed -i 's/3.0.0.dev0/2.2.0rc0/g'

# this updates versions in java files
grep -l -R "2.0.0-SNAPSHOT" * | grep -v "yaml" | grep -v "rst" | xargs -n 1 sed -i 's/2.0.0-SNAPSHOT/2.2.0rc0/g'
```